src/Makefile

```diff
@@ -77,11 +77,8 @@ else
 # JULIACODEGEN != LLVM
 endif
 
-RT_LLVM_LIBS := support
-
-ifeq ($(LLVM_VER_MAJ),16)
-RT_LLVM_LIBS += targetparser
-endif
+RT_LLVM_LIBS := support # for APMath and some other useful ADT
+RT_LLVM_LIBS += targetparser # for getHostCPUName on LLVM 16+
```

Comment:
I think we still support llvm 15? CC @vchuravy.

Reply:
I guess I could invert the condition, to be ifneq 15?

Reply:
```suggestion
ifneq ($(LLVM_VER_MAJ),15)
RT_LLVM_LIBS += targetparser # for getHostCPUName on LLVM 16+
endif
```